packages/approot/console/src/components/Launch/LaunchForm/DatetimeInput.tsx

```diff
@@ -8,7 +8,7 @@ import { getLaunchInputId } from './utils';
 
 /* This module imports differently in tests and causes errors.
  * The long-term solution is to enable `esmModuleInterop` and change all of
- * our `import * as` to be normal default imports. But as this is the only
+ * our `import` to be normal default imports. But as this is the only
  * module currently exhibiting the problem, this is a less impactful change.
  */
```

Comment:
I think this comment is no longer relevant - couldn't find any moment related imports with a * in tests

Reply:
oh yeah, find and replace. missed that one, thank you